Hunting Queries/SecurityEvent/Windows System Shutdown-Reboot(T1529)

```diff
@@ -0,0 +1,33 @@
+id: 024b3726-add7-4e06-842d-932034ba21f7
+name: Windows System Shutdown/Reboot(Sysmon)
+description: |
+  'This detection uses Sysmon telemetry to detect System Shutdown/Reboot (MITRE Technique: T1529)'
+
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Impact
+relevantTechniques:
+  - T1529
+
+query: |
+let timeframe = 1d;
+Event
+//This query uses sysmon data depending on table name used this may need updataing
+| where Source == "Microsoft-Windows-Sysmon"
+| extend RenderedDescription = tostring(split(RenderedDescription, ":")[0])
+| extend EventData = parse_xml(EventData).DataItem.EventData.Data
+| mv-expand bagexpansion=array EventData
+| evaluate bag_unpack(EventData)
+| extend Key=tostring(['@Name']), Value=['#text']
+| evaluate pivot(Key, any(Value), TimeGenerated, Source, EventLog, Computer, EventLevel, EventLevelName, EventID, UserName, RenderedDescription, MG, ManagementGroupName, Type, _ResourceId)
+| extend RuleName = column_ifexists("RuleName", ""), TechniqueId = column_ifexists("TechniqueId", ""),  TechniqueName = column_ifexists("TechniqueName", "")
+| parse RuleName with * 'technique_id=' TechniqueId ',' * 'technique_name=' TechniqueName
+| where EventID == 1
+| where Image contains "shutdown.exe" 
+| where CommandLine has_any ("/r /t" , "/s /t")
+| project UtcTime,User,Image, CommandLine
+| extend  User, Image, CommandLine
+
```

Comment:
Add in the Entity mapping information, see example [here](https://github.com/Azure/Azure-Sentinel/wiki/Contribute-to-Sentinel-GitHub-Community-of-Queries#example-yaml-schema-and-format-from-a-hunting-query).

Reply:
Thank you shainw for your great support. I will be waiting for your further reviews and comments